md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/util/Dao2TableUtil.java

```diff
@@ -26,8 +26,11 @@
  * Created by AR288503 on 1/2/2016.
  */
 public class Dao2TableUtil {
-    public static final Logger LOGGER = Logger.getLogger(Dao2TableUtil.class);
-    public static com.wipro.ats.bdre.md.beans.table.Process jpa2TableProcess(com.wipro.ats.bdre.md.dao.jpa.Process jpaProcess) {
+
+    private static final Logger LOGGER = Logger.getLogger(Dao2TableUtil.class);
+
+    private Dao2TableUtil(){}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)